python/cudf/cudf/api/extensions/accessor.py

```diff
@@ -1,13 +1,17 @@
 # Copyright (c) 2020, NVIDIA CORPORATION.
 
+import warnings
+
 import cudf
 from pandas.core.accessor import CachedAccessor
+from pandas.util._decorators import doc
```

Comment:
Given it's `_decorators` it makes me a bit uneasy about importing this from Pandas. Any other ideas?

Reply:
I would love to use `@docfmt` and `@copy_docstring` from `cudf.utils.docutils`. I played around and haven't figured out how to use it yet. Is there an example usage I can look it up?

Reply:
`@copy_docstring`: https://github.com/rapidsai/cudf/blob/555b91ae90e7900d2ebcd0d630de8f32dfd375a6/python/cudf/cudf/core/series.py#L1597-L1610

`docfmt_partial`: https://github.com/rapidsai/cudf/blob/671e8dd2cb9bda2381cf1f1822255d6f38ac2c8f/python/cudf/cudf/utils/ioutils.py#L19-L63